<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>apartmentPanel</class>
 <widget class="QDialog" name="apartmentPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>371</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lPicAdd">
          <property name="text">
           <string>Picture Address: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lPicAdd_2"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Total Area: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Number of Floors: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Number of Units: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Base Price: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Address: </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="lTArea"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lNFloors"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lNUnits"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lBasePrice"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lAddress"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Elevator</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>200</height>
       </size>
      </property>
      <property name="title">
       <string>Units: </string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>321</width>
         <height>161</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pAddUnit">
        <property name="text">
         <string>Add Unit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pShowUnit">
        <property name="text">
         <string>Show Unit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
